nss-day-cohort-44/rare-rest-api-malicous-vortices
# Added ability to toggle is_staff field if request comes from an admin

# ISSUE TICKET 
## Description


## Type of change


# Testing Instructions
`git fetch origin `
`git checkout ` 

Use the URL: http://localhost:8000/users/2/admin

Use an admin's auth token and send this request with a user code that is related to another user with a POST request.  Then do a get request for the same user and watch the is_staff status toggle.

Also try to send the request as a non-admin and make sure it won't let you toggle it.

EDIT: I have also now included the inability to remove the final admin

User Instructions: 

# Checklist:
